projection.tex

```diff
@@ -74,11 +74,138 @@
 %
 \end{abstract}
 
-\section{Projection Method}
 %
 \label{s:projection}
-
-TBA.
+\section{Govern equations for 2D incompressible flow}
+For simplicity, suppose $\rho=1$. \\
+Momentum equation in x-direction:
+\begin{align*}
+\frac{\partial u}{\partial t}+u \frac{\partial u}{\partial x}+v \frac{\partial u}{\partial y}=-\frac{\partial p}{\partial x}+ \nu (\frac{\partial^2 u}{\partial x^2}+\frac{\partial^2 u}{\partial y^2})
+\tag{1}
```

Comment:
Label should be used to allow symbolic reference.  See how it works in `cese.tex`.  I will create an issue for you to fix it.